### DISCUSSION TITLE
奥运会 - Miao Yu | 于淼

### DISCUSSION THREAD

**Discussion by giscus** [yufree/yufree.cn#473] [General]
# 奥运会 - Miao Yu | 于淼

https://yufree.cn/cn/2024/08/11/olympic/

**Comment by tcgriffith** [comment]
> 这个人数背景下在全球关注的奥运会开幕式里体现几分钟在我看来也没啥问题，你搞主旋律也没人看啊，导演这种项目怎么做都是错的...... 这里的核心问题在于，性少数群体跟喜欢吃甜豆腐脑还是咸豆腐脑群体都只是一种人群划分方法  

我觉得开幕式拉跨可不是性少数元素的问题, 单纯是因为变装皇后秀果体蓝精灵辣眼睛。 放一堆早年比利海灵顿那类肌肉男，正常人大概反应会是“虽然不懂这个调调 但是看起来很带感啊！”  深层次反映的本质是西海岸白左LGBT文化在米式霸权的加持下，对西欧传统大国训狗成果的展示。

> 赵高欲为乱，恐群臣不听，乃先设验，持鹿献于二世，曰：“马也。”二世笑曰：“丞相误邪？谓鹿为马。”问左右，左右或默，或言马以阿顺赵高。或言鹿，高因阴中诸言鹿者以法。后群臣皆畏高。

**Reply by yufree** [reply]
我是感觉无所谓，变装皇后啥的跟我生活也没交集，愿意发声就发。不过政治正确当前流行的深层原因我觉得是这东西可以纵向切隔人群找认同感与正义感，从而忽略掉按财富金字塔横向切带来的贫富不均矛盾。任何资本主导的现代国家政治人物都最喜欢通过某个不那么重要的议题分割人群找认同与对立，但又不敢触及贫富差距这个议题，因此只能在堕胎、枪支、毒品、性少数等议题上制造矛盾释放民众的权力欲望，不过是转移矛盾罢了。如果不推广这些议题把注意力放到贫富差距上，早就都打内战闹革命了。

**Comment by H-COOOH** [comment]
> 留给这个时代共同讨论话题的东西已经不多了。

为什么会出现这样的情况呢？是不是身处环境与接触信息的割裂最终导致的认知差异？

之前我读到有观点说社会正在趋于“原子化”，是否与这个现象有关呢？

**Reply by yufree** [reply]
单纯感慨，我这种老东西经历过全家听广播看电视倒背春晚小品台词的年代，那时候精神生活匮乏但人们共享有限信息源，交流时共同话题都是都知道的，现在各人有各人爱好与关注，不同年代人特别家人间共同关注就少很多了，也可以说家庭结构解体，很多话题已经是跨服聊天了，但另一方面也可以找到之前不可能联系到的同好，只是除非主动突破自己舒适区，大多人也懒得跟三观不同的人交流了。这只是现象，不涉及对错，保守回到过去是蠢，盲目认为自己“进步”是坏，少数人或者蠢或者坏，大多数人则是法律约束下既蠢又坏的个体暴君，始终找不到跟自己与世界的相处之道。
